interactive_ai/services/api/schemas/configuration/requests/revamp/configurable_parameters/bool.yaml

```
@@ -0,0 +1,16 @@
+type: object
+properties:
+  key:
+    type: string
+    description: Identifier for the boolean parameter used in code references
```

```suggestion
    description: Identifier for the boolean parameter
```
Not sure what you mean with "in code references", I would remove it